packages/esm-service-queues-app/src/hooks/useQueueEntries.ts

```diff
@@ -23,30 +23,31 @@ export function getInitialUrl(rep: string, searchCriteria?: QueueEntrySearchCrit
   const searchParam = new URLSearchParams();
   searchParam.append('v', rep);
   searchParam.append('totalCount', 'true');
- 
+
   if (searchCriteria) {
     for (let [key, value] of Object.entries(searchCriteria)) {
       if (value != null) {
         searchParam.append(key, value?.toString());
       }
     }
   }
- 
+
   return `${queueEntryBaseUrl}?${searchParam.toString()}`;
 }
 
+let queueEntryMutates: ReturnType<typeof useQueueEntries>['mutate'][] = [];
```

Comment:
Do we still need `useMutateQueueEntries` at all, or can we just have `useQueueEntries` return the `mutate` returned by `useOpenmrsFetchAll`?